jaxley/modules/base.py

```diff
@@ -840,6 +840,12 @@ def _gather_channels_from_constituents(self, constituents: List):
             name = channel._name
             self.base.nodes.loc[self.nodes[name].isna(), name] = False
 
+        # Set columns of groups to `False` instead of `NaN`.
+        for name in self.base.group_names:
+            self.base.nodes.loc[self.nodes[name].isna(), name] = False
+            # Ensure that type is boolean---in some cases, it had been an `object`.
+            self.base.nodes[name] = self.base.nodes[name].astype(bool)
```

Comment:
could we not save one for loop here?

i.e. 
```python
for name in in self.base.channels + self.base.pumps + self.base.group_names:
    self.base.nodes.loc[self.nodes[name].isna(), name] = False
    # Ensure that type is boolean---in some cases, it had been an `object`.
    self.base.nodes[name] = self.base.nodes[name].astype(bool)
```

Reply:
no, because channels need
```python
for channel in in self.base.channels
    name = channel._name
```